core/src/main/resources/hudson/model/Api/index.jelly

```diff
@@ -98,58 +98,47 @@ THE SOFTWARE.
         -->
       </dl>
 
-      <p>
-        For more information about remote API in Jenkins, see
-        <a href="https://jenkins.io/redirect/remote-api">the documentation</a>.
-      </p>
+      <p>For more information about remote API in Jenkins, see <a href="https://jenkins.io/redirect/remote-api">the documentation</a>.</p>
```

Comment:
@jglick I don't believe this is such case. This file has had (and has) very slow activity, I don't see the problem. At any case, changes related with the UI, are susceptible of resulting complex for reviewing, for that reason, I try to provide screenshots and manual tests.